app/components/quick-filter.js

```diff
@@ -70,7 +77,9 @@ export default Component.extend({
       this.setDateFilter();
       this.set('location', this.get('dummyLocation'));
       this.set('eventName', this.get('dummyName'));
-      this.set('disableClear', false);
+      if (this.get('dummyName', null) || this.get('dummyLocation', null)) {
```

Comment:
@kushthedude how does this work exactly. 
`this.get('dummyName', null)` what does it return ? Get is used to fetch properties. Not compare them.

Reply:
Also, this is the init hook. Read what it does in ember guides.

Reply:
I was using logical comparators but i was trying to do that thing with the .get like if this.get gets null then it should go .
I will change the logic if you say

Reply:
@kushthedude are you by any chance suggesting what you wrote works ?

Reply:
Im not suggesting it , i was just experimenting some random things , will make change ASAP

Reply:
Please do make sure to only push the experiments which you have 
 deemed successful :smile:

Reply:
@CosmicCoder96 I just checked right now this thing is also functioning as expected 😕

Reply:
@kushthedude why don't you just try to replace it with :
`if (this.dummyName === null) || this.dummyLocation === null)`
and see how it works?

@CosmicCoder96 I think this will be logically correct too?

Reply:
Welcome back @uds5501  😄 ; I know this will work i was just curious that the thing i did right now is working too 😕 . And yeah i pushed wrong commit by mistake , I was goin to use logical operators only😅

Reply:
@CosmicCoder96 I think what i wrote is working perfectly 😕

Reply:
@kushthedude , the function this.get takes in one argument not two. So what you have written is generating a garbage value which happens to return a false inside an if statement.
So you have to perform a comparison to get false logically not through an expression that generates a random undefined.

Reply:
Got it.